docs/ios/xcsync.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Xcsync for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcsync: improved support for Apple development via .NET
+
+Xcsync is a tool designed to provide developers using the .NET MAUI/.NET Apple SDKs have improved support for editing Apple specific files, such as storyboards and assets. The tool generates a temporary Xcode project from a .NET project, monitors both for changes, and synchronizes the changes between them.
+
+#### How to use Xcsync
+
+The tool has 2 commands right now: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET.<br />
+Xcsync can be invoked via a dotnet build target, like so:
+```dotnet build /t:xcsync-generate``` or ```dotnet build /t:xcsync-sync```
+
+You can also add the xcsync tool path to your PATH environment variable to invoke the tool directly as a more seamless dotnet command.
+
+You can customize several things regarding the tool's functionality like where the temporary Xcode project will be generated, if you would like to open the Xcode project upon being generated, how verbose you would like the logging output to be. Run ```xcsync [generate/sync] -h``` to learn more.
```

Comment:
This should be written for the `dotnet build /t:xcsync-*` commands, detailing each of the properties that can be set to override the defaults.